packages/google_sign_in/google_sign_in_web/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.12.4
+
+* Gets ready to update to web code to package `web: ^0.5.0`.
```

Comment:
This seems like an odd description since it actually does the update.

(Also, nit: this package is all web code, so doesn't need to specify that here.)

Reply:
Agh shoot, I did a grammar pass for this description in other packages, but this one wasn't an exact match and I forgot. Thanks for flagging this!

Reply:
I also made this small dependency change a `+Z` change; I'll bump `+Y` when the migration of `package:web` actually happens.